src/plugins/fftune_gui.c

```diff
@@ -134,6 +134,73 @@ static void fftune_gui_zoom_ofs (FFTuneGui *fftunegui, double zoom_amt,
 				 int zoom_xpos);
 static void fftune_gui_scroll_ofs (FFTuneGui *fftunegui, int index_ofs);
 
+#ifdef _WIN32
+/*
+ * Parse the first float found in a char array. Parsing is terminated
+ * when the end of string is reach or a non digit caracter is encountered.
+ * @param str char array to parse
+ * @return the first float detected in str or 0.0f.
+ * Example: 
+ * "123.2"     -> 123.2f
+ * "123.2text" -> 123.2f
+ * "text123.2" ->   0.0f
+ */
+static float 
+strtof(const char* str, char **endptr)
```

Comment:
Why do you need a custom `strtof()`? Can't you just use `strtod()` and cast the result to float?